regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_native_insert_stmt.groovy

```diff
@@ -0,0 +1,191 @@
+                `id` int(11) NOT NULL COMMENT "用户 ID",
+                `name` varchar(65533) NOT NULL DEFAULT "yixiu" COMMENT "用户姓名",
+                `score` int(11) NULL COMMENT "用户得分",
+                `test` int(11) NULL COMMENT "null test",
+                `dft` int(11) DEFAULT "4321")
+                UNIQUE KEY(`id`) DISTRIBUTED BY HASH(`id`) BUCKETS 1
+                PROPERTIES("replication_num" = "1", "enable_unique_key_merge_on_write" = "true")
+    """
+    sql """insert into ${tableName4} values(2, "doris2", 2000, 223, 1),(1, "doris", 1000, 123, 1),(3,"doris3",5000,34,345);"""
+    qt_4 """ select * from ${tableName4} order by id; """
+    sql "set enable_unique_key_partial_update=true;"
+    sql "set enable_insert_strict = false;"
+    sql "sync;"
+    // partial update with delete sign
+    sql "insert into ${tableName4}(id,__DORIS_DELETE_SIGN__) values(2,1);"
+    qt_4 """ select * from ${tableName4} order by id; """
+    sql "set enable_unique_key_partial_update=false;"
+    sql "sync;"
+    sql """ DROP TABLE IF EXISTS ${tableName4} """
+
+
+    def tableName5 = "test_partial_update_native_insert_stmt5"
+    sql """ DROP TABLE IF EXISTS ${tableName5} """
+    sql """
+            CREATE TABLE ${tableName5} ( 
+                `id` int(11) NULL, 
+                `name` varchar(10) NULL,
+                `age` int(11) NULL DEFAULT "20", 
+                `city` varchar(10) NOT NULL DEFAULT "beijing", 
+                `balance` decimalv3(9, 0) NULL, 
+                `last_access_time` datetime NULL 
+            ) ENGINE = OLAP UNIQUE KEY(`id`) 
+            COMMENT 'OLAP' DISTRIBUTED BY HASH(`id`) 
+            BUCKETS AUTO PROPERTIES ( 
+                "replication_allocation" = "tag.location.default: 1", 
+                "storage_format" = "V2", 
+                "enable_unique_key_merge_on_write" = "true", 
+                "light_schema_change" = "true", 
+                "disable_auto_compaction" = "false", 
+                "enable_single_replica_compaction" = "false" 
+            );
+    """
+    sql """insert into ${tableName5} values(1,"kevin",18,"shenzhen",400,"2023-07-01 12:00:00");"""
+    qt_5 """select * from ${tableName5} order by id;"""
+    sql "set enable_insert_strict = true;"
+    sql "set enable_unique_key_partial_update=true;"
+    sql "sync;"
+    // partial update using insert stmt in strict mode, the max_filter_ratio is always 0
+    test {
+        sql """ insert into ${tableName5}(id,balance,last_access_time) values(1,500,"2023-07-03 12:00:01"),(3,23,"2023-07-03 12:00:02"),(18,9999999,"2023-07-03 12:00:03"); """
+        exception "Insert has filtered data in strict mode"
+    }
+    qt_5 """select * from ${tableName5} order by id;"""
+    sql "set enable_unique_key_partial_update=false;"
+    sql "set enable_insert_strict = false;"
+    sql "sync;"
+    sql """ DROP TABLE IF EXISTS ${tableName5}; """
+
+    def tableName6 = "test_partial_update_native_insert_stmt6"
```

Comment:
We also need a case that have multiple key columns and user's statement only provide part of key columns.